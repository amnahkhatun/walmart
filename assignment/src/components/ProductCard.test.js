// import React from "react";
// import { shallow, mount, render } from "enzyme";
// import ProductCard from "./ProductCard";
// import "../setupTest.js";

// describe("ProductCard", () => {
//   it('should render correctly in "debug" mode', () => {
//     const component = shallow(<ProductCard />);

//     expect(component).toMatchSnapshot();
//   });
// });
