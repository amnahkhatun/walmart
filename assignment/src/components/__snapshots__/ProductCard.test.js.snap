// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductCard should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
